src/opcodes/opcodes.cairo

```diff
@@ -364,4 +365,26 @@ pub mod Opcode {
             _ => utils::not_implemented(ref engine)
         }
     }
+
+    pub fn is_opcode_disabled(opcode: u8, ref engine: Engine) -> Result<(), felt252> {
+        if opcode == 126
```

Comment:
Could you use the opcode names here now to make them easier to read? For example, this line would be something like `opcode == OP_CAT`.

Reply:
Sure!